src/Docfx.App/Helpers/PlaywrightHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using Docfx.Common;
+using Docfx.Exceptions;
+
+#nullable enable
+
+namespace Docfx;
+
+internal static class PlaywrightHelper
+{
+    public static void EnsurePlaywrightNodeJsPath()
+    {
+        // Skip if playwright environment variable exists.
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_SEARCH_PATH") != null)
+            return;
+
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_NODEJS_PATH") != null)
+            return;
+
+        if (!TryFindNodeExecutable(out var exePath, out var version))
+            throw new DocfxException("Node.js executable is not found. Try to install Node.js or set the `PLAYWRIGHT_NODEJS_PATH` environment variable.");
+
+        Logger.LogInfo($"Using custom Node.js {version} executable.");
+        Logger.LogVerbose($"Path: {exePath}");
```

Comment:
I've changed code to print Node.js version by default.
And exe path information is outputted as verbose log.